modules/sentinel/Makefile

```diff
@@ -0,0 +1,4 @@
+# Install hashicorp sentinel functions
+.PHONY: sentinel/hashicorp-functions
+sentinel/hashicorp-functions:
+	if [ ! -d "./sentinel-extensions" ]; then git submodule add https://github.com/hashicorp/terraform-sentinel-policies.git sentinel-extensions; fi
```

Comment:
I'm by no means an expert on this stuff but this would mean we pull in the whole repo right and not just the functions, meaning we could pull in their policies and our test and format pipeline steps pick them up? We wouldn't want this stuff failing for code we don't own. Is there a way to limit the submodule to just what we are interested in?

Reply:
The pipeline only runs against policies explicitly listed in sentinel.hcl in the root of our policy repo.